lib/credo_filename_consistency/check/consistency/filename_consistency.ex

```diff
@@ -99,7 +106,7 @@ defmodule CredoFilenameConsistency.Check.Consistency.FilenameConsistency do
     )
   end
 
-  def valid_filenames(module, root, extension, acronyms) when is_binary(module) do
+  defp valid_filenames(module, root, extension, acronyms) when is_binary(module) do
```

Comment:
Yes, thank you for this! I originally made the function public because I wanted to test it.